faq/domains-and-dns.mdx

```diff
@@ -0,0 +1,110 @@
+---
+meta:
+  title: Domains & DNS
+  description: Discover Domains & DNS.
+content:
+  h1: Domains & DNS
+  paragraph: Discover Domains & DNS.
+image: DNS.webp
+---
+
+## What are the prerequisites to register a Domain name?
+
+You need a valid and reliable email address and a valid debit or credit card. You must have a valid email address and make sure to keep your contact information updated during the whole period of registration.
+
+## Which rights are associated with the owner of a domain?
+
+The owner of a domain can update all the technical and administrative parameters referring to their domain name, indicate and change the administrative, technical and invoicing contacts. During the public beta, you must contact the support team to modify the contacts you have entered. 
```

Comment:
There is no "invoicing contact" in our product (because we actually are this contact on client behalf, and we invoice them in turn using our billing system)